libs/shared-entity/src/dto/workspace_dto.rs

```diff
@@ -121,7 +122,165 @@ pub struct CollabResponse {
   pub object_id: String,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct PublishedDuplicate {
+  pub published_view_id: String,
+  pub dest_view_id: String,
+}
+
+#[derive(Default, Debug, Clone, Serialize, Deserialize)]
+pub struct FolderView {
+  pub view_id: String,
+  pub name: String,
+  pub icon: Option<ViewIcon>,
+  pub is_space: bool,
+  pub is_private: bool,
+  pub extra: Option<serde_json::Value>,
```

Comment:
what is the extra used for? it would be better to have documentation here

Reply:
will add in some documentation here. AFAIK it currently holds information such as if the view is a space and font information